src/any_agent/callbacks/span_generation/google.py

```diff
@@ -0,0 +1,172 @@
+# mypy: disable-error-code="override,union-attr"
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING, Any
+
+from opentelemetry.trace import StatusCode, get_current_span
+
+from any_agent.callbacks.base import Callback
+from any_agent.callbacks.span_generation.common import _set_tool_output
+
+if TYPE_CHECKING:
+    from google.adk.agents.callback_context import CallbackContext
+    from google.adk.models.llm_request import LlmRequest
+    from google.adk.models.llm_response import LlmResponse
+    from google.adk.tools.base_tool import BaseTool
+    from google.adk.tools.tool_context import ToolContext
+    from opentelemetry.trace import Span, Tracer
+
+
+def _set_llm_output(llm_response: LlmResponse, span: Span) -> None:
+    content = llm_response.content
+    if not content:
+        return
+    if not content.parts:
+        return
+
+    if content.parts[0].text:
+        span.set_attributes(
+            {
+                "gen_ai.output": content.parts[0].text,
+                "gen_ai.output.type": "text",
+            }
+        )
+    else:
+        span.set_attributes(
+            {
+                "gen_ai.output": json.dumps(
+                    [
+                        {
+                            "tool.name": getattr(part.function_call, "name", "No name"),
+                            "tool.args": getattr(part.function_call, "args", "No args"),
```

Comment:
Maybe we could use a dict literal convention?

```suggestion
                            "tool.args": getattr(part.function_call, "args", "{}"),
```